content/en/tracing/visualization/service.md

```diff
@@ -41,69 +44,135 @@ Enable them directly or create your own [APM monitors][3].
 
 **Note**: Tag any monitor with `service:<SERVICE_NAME>` to attach it to an APM service.
 
+## Summary Cards
+
+The service page features summary cards with highlights on your service health. Easily spot potential faulty deployments, and click the card to view details or traces of the latest deployment, or view all deployments on this service. See new issues flagged on your service through our integration with [Error Tracking][9], where errors are automatically aggregated into issues.
+
+{{< img src="tracing/visualization/service/summary_cards.jpg" alt="Summary cards"  style="width:100%;">}}
+
+Our [Service Level Objectives (SLOs)][10] and [Incidents][11] summaries allow you to monitor the status of SLOs and ongoing incidents, so you can keep performance goals and top of mind. Click the cards to create a new SLO on the service or declare an incident.
```

Comment:
```suggestion
Our [Service Level Objectives (SLOs)][10] and [Incidents][11] summaries allow you to monitor the status of SLOs and ongoing incidents, so that you can keep performance goals top of mind. Click the cards to create a new SLO on the service or declare an incident.
```